documentation/assemblies/security/assembly-access-operator.adoc

```diff
@@ -0,0 +1,19 @@
+// This assembly is included in the following assemblies:
+//
+// security/assembly-security.adoc
+
+[id='assembly-access-operator-{context}']
+= Using the Access Operator to manage client connections (early access)
+
+[role="_abstract"]
+The Strimzi Access Operator simplifies how applications connect to a Kafka cluster managed by the Cluster Operator. 
+The Access Operator is deployed separately and manages the distribution of connection details, which are stored as a `Secret` resource. 
+The secret contains user credentials and cluster connection details, and can be referenced by applications across different namespaces.
```

Comment:
Later on in the text, the user credentials are defined as "optional" ... should we say the same here as well?